docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/20-advanced-with-argocd.md

```diff
@@ -4,6 +4,160 @@ sidebar_label: With Argo CD
 
 # Advanced Installation with Argo CD
 
+This section outlines the various advanced installation options available for deploying Kargo on a Kubernetes cluster using Argo CD.
+
+:::warning
+This section assumes that you have already installed any dependencies or prerequisites required for running Kargo on a Kubernetes cluster. Please see the [installation guide](../../40-operator-guide/10-basic-installation.md#prerequisites) for more information.
+:::
+
+## Argo CD Helm Application
+
+The most common way to deploy Kargo using Argo CD is to create a Helm `Application`, and use the `.spec.source.helm.parameters` section to specify any parameters you may need. This is the most straightforward way to deploy Kargo using Argo CD.
+
+:::info
+If using the `api.adminAccount.passwordHash` parameter, you must escape the `$` character with `$$` to prevent Helm from interpreting it as a variable. Please see [this discussion](https://discord.com/channels/1138942074998235187/1138946346217394407/1267966083168469102) for more information.
+:::
+
+```yaml
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: kargo
+  namespace: argocd
+spec:
+  project: default
+  destination:
+    namespace: kargo
+    server: https://kubernetes.default.svc
+  source:
+    repoURL: ghcr.io/akuity/kargo-charts
+    chart: kargo
+    targetRevision: 1.2.0
+    helm:
+      parameters:
+        - name: api.adminAccount.passwordHash
+          value: "$$2a$$10$$Zrhhie4vLz5ygtVSaif6o.qN36jgs6vjtMBdM6yrU1FOeiAAMMxOm"
+        - name: controller.logLevel
+          value: "DEBUG"
+        - name: api.adminAccount.tokenTTL
+          value: "24h"
+        - name: api.adminAccount.tokenSigningKey
+          value: "iwishtowashmyirishwristwatch"
+  syncPolicy:
+    automated:
+      prune: true
+      selfHeal: true
+    syncOptions:
+    - CreateNamespace=true
+```
+
+Conversely, insetad of using the `parameters` field under the `.spec.source.helm` section; you can use the `values` block or `valuesObject` object to specify the values for the Kargo Helm chart.
```

Comment:
```suggestion
Alternatively, instead of using the `.spec.source.helm` section's `parameters` field, you can use that same section's `values` or `valuesObject` fields to specify installation options.
```

Would it be worth adding an explicit example of this? tbh, I'd have to hit the Argo CD docs to figure this out, so most readers would probably find this to be a "cliff" as well.

Reply:
@krancour I thought about this too. I wanted to avoid providing an example for each implementation because it's really a matter of preference for the end user at that point (i.e. "no best practices here, do what works for you" sort of thing).

Maybe just link to the Argo CD docs that talks about these methods? There's *really* no pros/cons for these methods if that helps the decision any.

Reply:
I added an example